### PR TITLE
Kargo Bid Adapter: Updating tests and refactoring

### PR DESCRIPTION
**Pull request by njflynn**
## Type of change

- [x] Bugfix
- [x] Refactoring (no functional changes, no api changes)

## Description of change
Updating unit tests and removing erroneous package json changes.

### CODE REVIEW

**Review by jsadwith — APPROVED** [review]
Thanks Neil!

**Review by juliangan07 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -44,26 +44,26 @@
     "@wdio/sync": "~7.5.2",
     "ajv": "6.12.3",
     "assert": "^2.0.0",
-    "babel-loader": "^8.3.0",
+    "babel-loader": "^8.0.5",
```
Comment (marklw16) [inline package.json:47]: just an observation but noticed a good few module versions and going back to earlier versions. Is that intentional ?

Comment (jsadwith) [reply package.json:47]: Is this just reverting the previous package.json changes?

Comment (njflynn) [reply package.json:47]: Sorry @marklw16 only seeing your comment now. Yes this is intentional, during my work I inadvertently updates these versions and this PR is partly to bring them back in line with Prebid.js before we merge

**Review by marklw16 — APPROVED** [review]

**Pull request merged**